doc/running.rst

```diff
@@ -50,13 +53,17 @@ where ``<nest_install_dir>`` is the installation directory of NEST (e.g. ``/home
 
    nest.Install("nestmlmodule")
 
-PyNESTML is also available as a component and can therefore be used from within other Python tools and scripts. After PyNESTML has been installed, the following modules have to be imported:
+
+From Python
+-----------
+
+PyNESTML is also available as a Python package, and can therefore be used from within other Python tools and scripts. After PyNESTML has been installed, the following functions have to be imported:
 
 .. code-block:: python
 
    from pynestml.frontend.pynestml_frontend import to_nest, install_nest
 
-Subsequently, it is possible to call PyNESTML from other Python tools and scripts via:
+Subsequently, it is possible to call PyNESTML from other Python tools and scripts via calls to ``to_nest()``, which generates the C++ code for NEST, and ``install_nest()``, which compiles and builds the code into a NEST extension module. ``to_nest()`` can be called as follows:
 
 .. code-block:: python
```

Comment:
Github doesn't allow me to comment on line 72. `DEBUG` logging level should be added to the list.